TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -748,9 +748,9 @@ public class RoutingManager
                                 if (nextHasOutgoingConnections) {
```

Comment:
`nextHasOutgoingConnections` =>next lane has out going connection => next lane is source.